assets/component-slider.css

```
@@ -70,13 +70,25 @@ slider-component.slider-component-full-width {
+  .splide__track--draggable {
+    min-height: 100%;
+  }
+
+  .splide__arrows .splide__arrow {
+    background-color: white;
+    opacity: 1;
+  }
+
+  .splide__arrows--ttb .splide__arrow--next {
+    
+  }
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)